recipe/meta.yaml

```diff
@@ -1,67 +1,63 @@
-{% set version = "4.1.0" %}
+{% set name = "altair" %}
+{% set version = "5.0.1" %}
+
 
 package:
-  name: altair
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/a/altair/altair-{{ version }}.tar.gz
-  sha256: 3edd30d4f4bb0a37278b72578e7e60bc72045a8e6704179e2f4738e35bc12931
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/altair-{{ version }}.tar.gz
+  sha256: 087d7033cb2d6c228493a053e12613058a5d47faf6a36aea3ff60305fd8b4cb0
+
 build:
-  number: 1
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
+    - hatchling
+    - wheel
   run:
-    - python >=3.6
-    - entrypoints
+    - python
+    - importlib-metadata  # [py<38]
     - jinja2
-    - jsonschema
-    - numpy >=0.18
-    - pandas
+    - jsonschema >=3.0
+    - numpy
+    - pandas >=0.18
     - toolz
+    - typing-extensions >=4.0.1  # [py<311]
 
 test:
+  imports:
+    - altair
+    - altair.expr
+    - altair.utils
+    - altair.vegalite
   source_files:
     - altair
   requires:
     - pytest
     - sphinx
     - docutils
```

Comment:
```suggestion
```
sphinx and docutils are for building documentation

Reply:
Done